docs/development.md

```diff
@@ -0,0 +1,101 @@
+# Development Guide
+
+## Building OpenCompose
+
+Read about building OpenCompose [here](https://github.com/redhat-developer/opencompose#go-from-source).
+
+## Workflow
+### Fork the main repository
+
+1. Go to https://github.com/redhat-developer/opencompose
+2. Click the "Fork" button (at the top right)
+
+### Clone your fork
+
+The commands below require that you have $GOPATH. We highly recommended you put OpenCompose code into your $GOPATH.
+
+```console
+git clone https://github.com/$YOUR_GITHUB_USERNAME/opencompose.git $GOPATH/src/github.com/redhat-developer/opencompose
```

Comment:
Well, it's not required even though it works. The general idea is the we directly copy from the URL in the browser and do a `git clone`, and the URL does not have a `.git`.